frontend/components/layout/Footer.vue

```diff
@@ -54,6 +54,11 @@ export default Vue.extend({
             validator: type => ['default', 'unified'].includes(type),
         },
     },
+    data() {
+        return {
+            tacPath: './pdf/Unified-Auction-UI_Terms.Conditions.pdf',
+        };
+    },
```

Comment:
> Please never hardcode the same variable in two places, as the next time we need to update the terms, we would forgot to edit one of them

Where do you suggest setting this then? As a prop doesn't really work since the two components (Footer, TermsModal) don't have a shared parent. As an `env variable`?